docs/adr/0016-pr-review.md

```diff
@@ -0,0 +1,64 @@
+# Reviewing Pull Requests
+
+* Status: accepted
+* Deciders: tumido, HumairAK, lars, goern, 4n4nd
+* Date: 2021-05-20
+
+Technical Story: [#93](https://github.com/operate-first/blueprint/issues/93)
+
+## Context and Problem Statement
+
+Each community has its own `contributing.md` file which outlines how are users expected to contribute to the community. That guide for contributions should also clearly state what will happen once a PR is submitted and how reviews are performed.
```

Comment:
Well..we don't have it in every repo yet, that in it self is also a problem. Maybe we should add it to all the repos  and as part of the template too?

Reply:
Yeah, we probably should.

Reply:
Actually, @HumairAK I don't feel like we should replicate content that much. Would you mind if I just transfer the contributing doc into blueprints as an ADR and then create a blank contributing doc in each repo linking to that contributing ADR?

Reply:
yeah that's fine